nautobot/ipam/api/views.py

```diff
@@ -53,13 +53,8 @@ class NamespaceViewSet(NautobotModelViewSet):
 
 
 class VRFViewSet(NautobotModelViewSet):
-    queryset = (
-        VRF.objects.select_related("tenant").prefetch_related("import_targets", "export_targets", "tags")
-        # FIXME(jathan): See if we need to revise the counts for prefixes/ips, here?
-        # .annotate(
-        #     ipaddress_count=count_related(IPAddress, "vrf"),
-        #     prefix_count=count_related(Prefix, "vrf"),
-        # )
+    queryset = VRF.objects.select_related("namespace", "tenant").prefetch_related(
```

Comment:
I'm guessing we need some more automatic select_related/prefetch_related code to handle UIViewsets?

Reply:
Not asking you to do that, just making sure I understand why this wasn't removed.

Reply:
This is the API views file - we don't yet have automatic code in that case, only the UI ListViews and similar.